pkg/executor/kopia/maintenance.go

```diff
@@ -213,6 +238,19 @@ func runMaintenance(maintenanceType string) error {
 	return nil
 }
 
+func returnDirList(parentDir string) ([]string, error) {
+	var files []string
+	fileInfo, err := ioutil.ReadDir(parentDir)
+	if err != nil {
+		return files, err
+	}
+
+	for _, file := range fileInfo {
+		files = append(files, file.Name())
```

Comment:
```suggestion
		if file.IsDir() {
			files = append(files, file.Name())
		}
```
Return only the directories and not the files inside it

Reply:
@kesavan the ReadDir API will not return file entries rather it will return directory entries all the time under the directory that is passed to the call. hence this check wont be necessary . Moreover we only create these directories under generic backup hence there won't be a situation to get a file entry here.